src/main/groovy/life/qbic/flowstore/database/MariaDBStorage.groovy

```diff
@@ -164,7 +165,7 @@ class MariaDBStorage implements Workflows, AutoCloseable{
 
     private Integer storeRunInfo(RunInfo runInfo) {
         def primaryKey
-        if( isRunInfoStored(runInfo) ) {
+        if (isRunInfoStored(runInfo)) {
```

Comment:
the bracket belongs syntactically to the `if` condition, therefore i try to do spaces like this